Sources/LCLSpeedtest/SpeedTestable.swift

```diff
@@ -21,7 +21,7 @@ import NIOWebSocket
 /// Implementation of this protocol need to accept an URL, which is the URL of the test server
 public protocol SpeedTestable {
 
-    init(url: URL)
+    init(url: URL, deviceName: String?)
```

Comment:
maybe we should not change the designated initializer here. Instead make a convenience initializer to accept `deviceName`